src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
 /* Small tablet (600px) */
 @media only screen and (min-width: 37.5em) {
+
+    th {
+        width: 70px;
```

Comment:
It looks better when it's consistent across devices sizes. That said, I've updated it to ems.